docs-app/app/components/feature-card.gts

```diff
@@ -0,0 +1,20 @@
+import type { TOC } from '@ember/component/template-only';
+
+const FeatureCard: TOC<{
+  Element: HTMLDivElement;
+  Args: { title: string; }
+  Blocks: {
+    default: [];
+  }
+}> = <template>
+  <div ...attributes>
+    <h4 class="pb-4 type-2xl text-titles-and-attributes">
+      {{@title}}
+    </h4>
+    <p class="text-body-and-labels">
+      {{yield}}
+    </p>
+  </div>
+</template>;
+
+export default FeatureCard;
```

Comment:
Maybe we can/should move these over to `oss-docs` eventually? (Not part of this PR, obvs, low priority!)

Reply:
Yeah, I think this was even mentioned somewhere, in a comment or so. Just vaguely remember...